README.md

```diff
@@ -301,6 +301,10 @@ cd aws
 go test -v -run TestListAMIs
 ```
 
+### Building and running in Docker
+
+In order to build and run as a docker container, have a look at the instructions at the top of the Dockerfile in this repository.
```

Comment:
NIT: Any reason to have the docs in the `Dockerfile` rather than the README? Seems like it would be a bit easier to read in a README with Markdown rendering, syntax highlighting, etc.

Reply:
I guess i just started with the Dockerfile and only added the read me after. I'm happy to change it, and move docs to the readme.